### ISSUE TITLE
Cannot open shared object file libMagickWand-6.Q16.so

### ISSUE THREAD

**Issue by zygmuntszpak** [JunoLab/Juno.jl#126]
Executing the following commands inside the Juno Julia console
```
using Images, ImageView,TestImages
img = testimage("mandrill")
```
yields the error:
```
Error encountered while loading "/home/zygmunt/.julia/v0.6/TestImages/images/mandrill.tiff".
Fatal error:
ERROR: InitError: could not load library "/usr/lib/x86_64-linux-gnu/libMagickWand-6.Q16.so"
/usr/lib/x86_64-linux-gnu/libMagickWand-6.Q16.so: cannot open shared object file: No such file or directory
during initialization of module ImageMagick
```
I am running the following version of Julia
```
   _       _ _(_)_     |  A fresh approach to technical computing
  (_)     | (_) (_)    |  Documentation: https://docs.julialang.org
   _ _   _| |_  __ _   |  Type "?help" for help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 0.6.2 (2017-12-13 18:08 UTC)
 _/ |\__'_|_|_|\__'_|  |  Official http://julialang.org/ release
|__/                   |  x86_64-pc-linux-gnu

```
with 
* Atom  0.6.11
* Julia Client 0.6.10
* Ink 0.8.3
* Uber Juno 0.1.2

If I execute the aforementioned commands from a Julia REPL outside of Juno everything works fine. I've also confirmed that everything works correctly using the VSCode editor. Hence, I suspect that something in Juno is causing the problem.

**Comment by pfitzseb** [comment]
Atom sometimes does weird things to your `PATH`, which probably is the reason for this. So far I couldn't repro your issue, but I also don't have access to my linux box right now. 
It would be very helpful if you could compare `ENV` in Juno and in the REPL.

**Comment by zygmuntszpak** [comment]
You are right, the problem is that the PATH variable is different. Shelling out from the Julia terminal inside Atom gives:

 `PATH=/home/zygmunt/.var/app/io.atom.Atom/data/python/bin:/app/bin:/usr/bin (ATOM)`

whereas shelling out from the canonical Julia terminal gives:

`PATH=/home/zygmunt/bin:/home/zygmunt/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games`

Do you happen to know how to specify the PATH for Atom?

**Comment by pfitzseb** [comment]
You're probably setting your `PATH` in `.bashrc` or something similar, right? In that case you should get the correct `PATH` when starting Atom from your shell.
